.php_cs.dist

```diff
@@ -121,6 +120,33 @@ return Config::create()
         'strict_comparison' => true,
         'yoda_style' => false,
         'single_line_throw' => false,
+        'compact_nullable_typehint' => true,
+        'final_static_access' => true,
+        'fully_qualified_strict_types' => true,
+        'global_namespace_import' => [
```

Comment:
Do we really want to enable this fixer?

Reply:
For me yes on all levels:

- Importing a symbol is a good practice, instead of relying on a dark fallback mechanism
- It is a performance improvement
- It is more readable than using leading backslashes occasionally (as we are using only for the opcached optimized functions)
- It makes the symbol resolution easier, cf. PHP-Scoper & static analysis tools
- It is in the folded import statements: it doesn't hurt readability at least not more than the huge license header